apps/ios/launchPackager.command

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Set terminal title
+echo -en "\\033]0;Metro\\a"
+clear
+
+THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
+
+# shellcheck source=/dev/null
```

Comment:
Can probably delete this comment?

Reply:
Actually is shellcheck clarifying potential errors like how we comment above lines to disable linting errors? Would be great to understand what that's for

Reply:
This is copied from the RN version... so I have no idea what this line is supposed to mean :D
I can trim it down to the necessary bits

Reply:
Decided to keep this in just so we don't have too many differences between our custom file and the react-native one. Also for education purposes if we ever decide to make a proper upstream fix where we don't need a custom commmand file anymore